src/sentry/utils/sdk_crashes/README.rst

```diff
@@ -0,0 +1,35 @@
+SDK Crash Detection
+-------
+
+
+Background
+=======
+
+As an APM company, the reliability of our SDKs is one of our most essential quality goals. If our SDK breaks the customer, we fail.
+Our SDK philosophy refers to this as `degrade gracefully <https://develop.sentry.dev/sdk/philosophy/#degrade-gracefully>`_.
```

Comment:
```suggestion
Our SDK philosophy refers to this as `degrade gracefully <https://develop.sentry.dev/sdk/philosophy/#degrade-gracefully>`.
```
?

Reply:
I think you need the `_` for external links in `.rst`, see https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages.